runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/BossAbbreviations.java

```diff
@@ -0,0 +1,83 @@
+package net.runelite.client.plugins.chatcommands;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BossAbbreviations {
```

Comment:
Is it important for this to be in a separate file? This could be a constant in the chat commands file, like it effectively was before.

Reply:
Also, do not make this `public`. This should be package-private at most.

Reply:
IMO they should be in their own file, just like `SkillAbbreviations`.

Reply:
I don't see why it's in a map in the first place. And if it's not being used elsewhere, it should even be split out.